torch/csrc/jit/import.h

```diff
@@ -17,28 +17,34 @@ namespace jit {
 using ModuleLookup = std::function<std::shared_ptr<script::Module>(
     const std::vector<std::string>&)>;
 
+static script::ExtraFilesMap default_extra_files;
```

Comment:
might be easier to just pass pointer instead of reference?

Reply:
Pointers are messy in pybind (needs to be smart pointer).